src/styleguide/_patterns/03-organisms/00-page-structure/01-page-footer.mustache

```diff
@@ -1,3 +1,12 @@
 <footer role="contentinfo">
   <p>{{footer.copy}} <a href="{{{url}}}">{{> atoms-logotype(label: "buildit @ wipro digital") }}</a></p>
-</footer>
\ No newline at end of file
+  <ul class="grav-c-list-horizontal">
+    {{#footer.social}}
+      <li>
+        <a href="{{url}}">
+          {{> atoms-inline-svg-symbol:grav-c-icon(symbolId: '{{icon}}', width: 100, height: 100 label: {{label}}) }}
```

Comment:
Why not just include `atoms-icon`? That would save you needing to manually set the class name and also make Pattern Lab display a link to that atom in the footer's "pattern info" panel.